### PR TITLE
Allow cupy.copyto from python scalar

### PR DESCRIPTION
**Pull request by niboshi**
Fixes https://github.com/pfnet/chainer/issues/720

### CODE REVIEW

File: tests/cupy_tests/manipulation_tests/test_basic.py [hunk 1]
```diff
@@ -52,3 +54,35 @@ def test_copyto_multigpu(self, xp, dtype):
...
+         'dst_dtype': testing.helper._make_all_dtypes(False, False)}))
+@testing.gpu
+class TestCopytoFromScalar(unittest.TestCase):
+
+    def _do_copyto(self, xp):
+        dst = xp.ones(self.dst_shape, dtype=self.dst_dtype)
+        xp.copyto(dst, self.src)
+        return dst
+
+    def test_copyto(self):
```
Comment (okuta) [inline tests/cupy_tests/manipulation_tests/test_basic.py]: Can we use `cupy.testing.numpy_cupy_allclose` for simplify this code?

File: cupy/manipulation/basic.py [hunk 2]
```diff
@@ -21,12 +22,27 @@ def copyto(dst, src, casting='same_kind', where=None):
...
+
+    if not can_cast:
         raise TypeError('Cannot cast %s to %s in %s casting mode' %
-                        (src.dtype, dst.dtype, casting))
+                        (src_dtype, dst.dtype, casting))
     if dst.size == 0:
         return
 
+    if src_is_python_scalar:
+        dst[...] = src
```
Comment (okuta) [inline cupy/manipulation/basic.py]: How about using `cupy.fill`?

**Comment by niboshi** [comment]
I didn't know `testing.numpy_cupy_allclose` has `accept_error` argument!
Fixed according to your review.

**Comment by okuta** [comment]
OK!

**Pull request merged**